doc/manual/parallel-computing.rst

```diff
@@ -451,36 +550,33 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
-Channels
```

Comment:
What is the API for a `RemoteTask` ? An Erlang style messaging any task in the cluster? Named tasks? What do you have in mind?